builder/build_server.py

```diff
@@ -55,21 +55,26 @@
         for message in queue.receive_messages():
             msg = json.loads(message.body)
             logger.info(f"Build server received SQS message {msg}")
-            if set(msg.keys()) == {"model_id", "s3_uri"}:
+            if set(msg.keys()) <= {"model_id", "s3_uri", "endpoint_only"}:
```

Comment:
I added `endpoint_only` because we need to be able to distinguish between the cases when we are putting up a model from scratch (from dynalab) or from being takendown. 

Alternatively, we can just check the model status here and then change the input to `deployer.deploy` instead of changing the structure of the build queue messages (old messages should still work, since `set(msg.keys()) <= {"model_id", "s3_uri", "endpoint_only"}` checks if `msg.keys()` is a subset of `{"model_id", "s3_uri", "endpoint_only"}` (but then there could be messages with just `model_id` or just `s3_uri` that would cause the build server to crash).